test/e2e/run_signal_test.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 const sigCatch = "trap \"echo FOO >> /h/fifo \" 8; echo READY >> /h/fifo; while :; do sleep 0.25; done"
+const sigCatch2 = "trap \"echo Received\" 8; while :; do sleep 0.25; done"
```

Comment:
Dumb question: Does the trap signal (`8`) need to change as well?

Reply:
nvm, `8 == SIGFPE`.  hmmm.  Though dumping core is the default action there, where as SIGPIPE is just process termination.